api/src/handlers.rs

```diff
@@ -596,5 +598,5 @@ fn generate_asset_name() -> String {
         .take(40)
         .map(char::from)
         .collect();
-    return format!("{}_{}.kml", Utc::now().timestamp(), rand_string);
+    format!("{}_{}.kml", Utc::now().timestamp(), rand_string)
```

Comment:
is this the prefered way to do it in rust?

Reply:
Yes, in Rust, Functions always return their last expression by default. `return` should only be used for early returns.